core_api/src/routes/chat.py

```diff
@@ -162,3 +168,125 @@ def rag_chat(chat_request: ChatRequest, _user_uuid: Annotated[UUID, Depends(get_
         for langchain_document in result.get("input_documents", [])
     ]
     return ChatResponse(output_text=result["output_text"], source_documents=source_documents)
+
+
+@chat_app.websocket("/rag-stream")
+async def websocket_endpoint(websocket: WebSocket):
+    await websocket.accept()
+
+    prompt_search_query = ChatPromptTemplate.from_messages(
+        [
+            MessagesPlaceholder(variable_name="chat_history"),
+            ("user", "{input}"),
+            (
+                "user",
```

Comment:
Is this a system prompt?

Reply:
(this question might be moot when integrating with existing code from redbox/llm/prompts/chat.py )